### ISSUE TITLE
[Umbrella-Item] Planned "async feature" work for .NET 10

### ISSUE THREAD

**Issue by KlausLoeffelmann** [dotnet/winforms#10739]
Prio 1:
- [x] #4631
- [x] #5917

Prio 2:
- [ ] #8582 
- [ ] #8547 

Prio 2, but may slip to 10.
- [ ] #5918

**Comment by elachlan** [comment]
@KlausLoeffelmann You have #4631 in Prio 1, but also in "Prio 2, but may slip to 10". Did you tag the issue right?

**Comment by KlausLoeffelmann** [comment]
No, I did not. That was the wrong link - correcting! Good catch, thanks!
